lib/src/booleans/boolean_touches.dart

```diff
@@ -0,0 +1,589 @@
+import '../../helpers.dart';
+import 'boolean_point_in_polygon.dart';
+import 'boolean_point_on_line.dart';
+
+/// Boolean-touches [true] if none of the [Point]s common to both geometries
+/// intersect the interiors of both geometries.
+/// example
+/// ```dart
+/// var line = LineString(coordinates;[Positin.of([1, 1]), Positin.of([1, 2]), Positin.of([1, 3]), Positin.of([1, 4])]);
+/// var point = Point(coordinates: Positon.of([1, 1]));
+/// booleanTouches(point, line);
+/// //=true
```

Comment:
the code block is not closed ```

Reply:
this is important, because otherwise I won't be able to review the code properly, syntax highlighting is disabled